doc/api/dgram.md

```diff
@@ -299,6 +299,84 @@ added: v0.6.9
 Sets or clears the `SO_BROADCAST` socket option.  When set to `true`, UDP
 packets may be sent to a local interface's broadcast address.
 
+### socket.setMulticastInterface(multicastInterface)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `multicastInterface` {String}
+
+*Note: All references to scope in this section are refering to
+[IPv6 Zone Indices][], which are defined by [RFC 4007][]. In string form, an IP
+with a scope index is written as `'IP%scope'` where scope is an interface name or
+interface number.*
+
+Sets the default outgoing multicast interface of the socket to a chosen
+interface or back to system interface selection. The `multicastInterface` must
+be a valid string representation of an IP from the socket's family.
+
+For IPv4 sockets, this should be the IP configured for the desired physical
+interface. All packets sent to multicast on the socket will be sent on the
+interface determined by the most recent successful use of this call.
+
+For IPv6 sockets, `multicastInterface` should include a scope to indicate the
```

Comment:
I've no idea in terms of informal English. In [RFC terminology](https://www.ietf.org/rfc/rfc2119.txt), Line 315 has the only actual MUST, leaving some strong recommendations that I would call SHOULD:

In the IPv4 case, I mean SHOULD in the sense that any OS MAY be choosing other behaviors for other IPs that one should not rely on for maximum portability.
In the IPv6 case, I mean SHOULD as in it is technically optional to add the default scope, but I would recommend including it specifically to avoid bugs through miscommunication.